README.md

```diff
@@ -75,15 +75,28 @@ The resulting binary is `bin/neo-go.exe`.
 
 ## Running a node
 
-A node needs to connect to some network, either local one (usually referred to
-as `privnet`) or public (like `mainnet` or `testnet`). Network configuration
-is stored in a file and NeoGo allows you to store multiple files in one
-directory (`./config` by default) and easily switch between them using network
-flags.
+Before running a Neo node, you need to generate the necessary configuration files
+for the network you wish to connect to, such as a local network (`privnet`), or 
+a public network like `mainnet` or `testnet`. NeoGo stores network configuration 
+files in a directory (`./config` by default) and allows you to easily switch 
```

Comment:
`NeoGo allows you to store multiple files in one directory` - this text shouldn't be removed.